nixos/modules/services/databases/postgresql.nix

```diff
@@ -22,6 +22,16 @@ let
       ${cfg.extraConfig}
     '';
 
+  ensureDatabasesOpts = {
```

Comment:
@petabyteboy you might be interested in joining our [discussion](https://github.com/NixOS/nixpkgs/pull/68353#issuecomment-541994707) about whether extending the `mysql` and `postgresql` database provisioning options is appropriate for NixOS or not.

Reply:
I see the point of that discussion, but to be honest I don't care. I don't need this functionality, I just added it because people asked for it and I will happily remove it when there's a consensus that we don't want this kind of functionality.